src/pages/description-page/DescriptionPage.tsx

```diff
@@ -13,12 +15,16 @@ export const DescriptionPage = ({
 
   return (
     <>
-      <h1 className="govuk-heading-l govuk-!-margin-top-6">
+      <h1 className="govuk-heading-l govuk-!-margin-top-6 govuk-!-margin-bottom-3">
         Local Plan description
         <span className="govuk-caption-m govuk-!-margin-top-3">
           Keep this specific to your own Local Plan
         </span>
       </h1>
+      <Details
```

Comment:
I need to tweak this a little bit as it overlaps with the error highlight when validation fails
![image](https://github.com/digital-land/local-plans-timetable/assets/98337074/74c343f8-d4b6-45b2-9f1e-278cb722248a)

Reply:
sorted :)